onnxruntime/core/framework/library_handles.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <utility>
+#include "core/platform/env.h"
+
+namespace onnxruntime {
+
+/**
+  * Unloads dynamic library handles loaded via Env::LoadDynamicLibrary() upon destruction.
```

Comment:
>Unloads dynamic library handles loaded via Env::LoadDynamicLibrary() upon destruction. [](http://example.com/codeflow?start=4&length=86)

We need more documentation here, as to where it is used and its purpose

Reply:
Added more information to comment.